### DISCUSSION TITLE
Update v0.104 to v0.106 in Debian ?

### DISCUSSION THREAD

**Discussion by derf81** [AdguardTeam/AdGuardHome#2775] [Q&A]
Hello,

I have a virtual machine running Debian 10 with AdGuard Home version: v0.104.3-SNAPSHOT-641db73a installed.

I have a message at the top that tells me AdGuard Home v0.106.1-SNAPSHOT-91403d0b is available! Click here for more information.

Could you tell me how can I do to update to v0.106.1? (in command line under debian or from a menu in AdGuard Home)

When I check the updates with the icon at the bottom right of the installed version it tells me that my version is up to date.

Thank you for your help

**Comment by ameshkov** [comment]
According to how the message looks like, you're on the "edge" update channel.

I am not sure why it does not suggest an automatic update, maybe you're not running it under root?

Anyway, the manual update is simple:
1. Download the latest edge build: https://static.adguard.com/adguardhome/edge/AdGuardHome_linux_amd64.tar.gz
2. Unpack it and replace AdGuard Home binary
3. Then simply restart AdGuard home

**Reply by Kratosak** [reply]
Does the same apply for Raspberry PI 4 update?
